src/backend/expungeservice/charge_classifier.py

```diff
@@ -316,6 +318,11 @@ def _traffic_crime(statute, name, level, disposition):
         if "infraction" in level:
             return AmbiguousChargeTypeWithQuestion([TrafficViolation()])
 
+    @staticmethod
+    def _contempt_of_court(name):
+        if "contempt of court" in name:
```

Comment:
@michaelzhang43 Is it sometimes just "contempt" or is it always fully written out in OECI as "contempt of court"? Or to put it differently, is there any way that "contempt" can be in the name without it being a "contempt of court"?

Reply:
It's usually Contempt of Court, but I strongly believe there are no other criminal or civil charges with the word "Contempt", so Ok to assume that as well.